runtime/cere/constants/src/lib.rs

```diff
@@ -63,7 +63,7 @@ pub mod time {
 	// 1 in 4 blocks (on average, not counting collisions) will be primary BABE blocks.
 	pub const PRIMARY_PROBABILITY: (u64, u64) = (1, 4);
 
-	pub const EPOCH_DURATION_IN_BLOCKS: BlockNumber = 4 * HOURS;
+	pub const EPOCH_DURATION_IN_BLOCKS: BlockNumber = 5 * MINUTES;
```

Comment:
I think this is a wrong value to set for the production runtime.

Reply:
I think this PR should be closed as these short timings were added for temporal testing purposes